src/components/Circles/ProfileImage.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useCallback, useState } from 'react';
+import { ViewStyle } from 'react-native';
+import { Avatar } from 'react-native-paper';
+import { initials } from './initials';
+import type { Post } from '../../hooks/Circles/types';
+import { IconSource } from 'react-native-paper/lib/typescript/src/components/Icon';
+import { createStyles } from '../BrandConfigProvider';
+import { useStyles } from '../BrandConfigProvider/styles/StylesProvider';
+import { tID } from '../TrackTile/common/testID';
+
+type Props = {
+  post: Post;
+  size: number;
+  fallbackIcon: IconSource;
+  style?: ViewStyle;
+};
+
+export const ProfileImage = ({ post, size, fallbackIcon, style }: Props) => {
+  const { styles } = useStyles(defaultStyles);
+  const [imageError, setImageError] = useState(false);
+  const profileInitials = useCallback(
+    () => (
+      <Avatar.Text
+        testID={tID('profile-image')}
+        size={size}
+        style={style}
+        label={initials(post?.author?.profile?.displayName)}
+      />
+    ),
+    [post?.author?.profile?.displayName, size, style],
+  );
+
+  if (post.author?.profile.picture) {
```

Comment:
This could just be `if (post.author?.profile.picture && !imageError) {` which would remove the ternary logic and not require the initial Avatar extraction.

Reply:
It's so obvious now. I'll update it.

Reply:
Done